docs/reference-guides/data/data-core-blocks.md

```diff
@@ -717,231 +717,4 @@ _Returns_
-
-_Returns_
-
--   `Object`: Action object.
-
-### removeBlockVariations
-
-Returns an action object used in signalling that block variations have been removed.
-
-_Parameters_
-
--   _blockName_ `string`: Block name.
--   _variationNames_ `string|string[]`: Block variation names.
-
-_Returns_
-
--   `Object`: Action object.
-
-### setCategories
-
-Returns an action object used to set block categories.
-
-_Usage_
-
-```js
-import { store as blocksStore } from '@wordpress/blocks';
-import { useSelect, useDispatch } from '@wordpress/data';
-
-const ExampleComponent = () => {
-	const blockCategories = useSelect(
-		( select ) => select( blocksStore ).getCategories(),
-		[]
-	);
-
-	const { setCategories } = useDispatch( blocksStore );
-
-	return (
-		<Button
-			onClick={ () =>
-				setCategories( [
-					...blockCategories,
-					{ title: 'Custom Category', slug: 'custom-category' },
-				] )
-			}
-		>
-			{ __( 'Add a new custom block category' ) }
-		</Button>
-	);
-};
-```
-
-_Parameters_
-
--   _categories_ `WPBlockCategory[]`: Block categories.
-
-_Returns_
-
--   `Object`: Action object.
-
-### setDefaultBlockName
-
-Returns an action object used to set the default block name.
-
-_Usage_
-
-```js
-import { store as blocksStore } from '@wordpress/blocks';
-import { useDispatch } from '@wordpress/data';
-
-const ExampleComponent = () => {
-	const { setDefaultBlockName } = useDispatch( blocksStore );
-
-	return (
-		<Button onClick={ () => setDefaultBlockName( 'core/heading' ) }>
-			{ __( 'Set the default block to Heading' ) }
-		</Button>
-	);
-};
-```
-
-_Parameters_
-
--   _name_ `string`: Block name.
-
-_Returns_
-
--   `Object`: Action object.
-
-### setFreeformFallbackBlockName
-
-Returns an action object used to set the name of the block used as a fallback
-for non-block content.
-
-_Parameters_
-
--   _name_ `string`: Block name.
-
-_Returns_
-
--   `Object`: Action object.
-
-### setGroupingBlockName
-
-Returns an action object used to set the name of the block used
-when grouping other blocks
-eg: in "Group/Ungroup" interactions
-
-_Parameters_
-
--   _name_ `string`: Block name.
-
-_Returns_
-
--   `Object`: Action object.
-
-### setUnregisteredFallbackBlockName
-
-Returns an action object used to set the name of the block used as a fallback
-for unregistered blocks.
-
-_Parameters_
-
--   _name_ `string`: Block name.
-
-_Returns_
-
--   `Object`: Action object.
-
-### updateCategory
-
-Returns an action object used to update a category.
-
-_Parameters_
-
--   _slug_ `string`: Block category slug.
--   _category_ `Object`: Object containing the category properties that should be updated.
-
-_Returns_
-
--   `Object`: Action object.
-
-<!-- END TOKEN(Autogenerated actions|../../../packages/blocks/src/store/actions.js) -->
+The actions in this package shouldn't be used directly. Instead, use the functions listed in the public API [here](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-blocks/)
```

Comment:
According to contributing guides https://github.com/WordPress/gutenberg/tree/trunk/docs/contributors/documentation#using-links this link should look as follows:

```suggestion
The actions in this package shouldn't be used directly. Instead, use the functions listed in the public API [here](/packages/blocks/README.md)
```

The rest will magically happen behind the scenes.